src/array_api_stubs/_draft/linalg.py

```diff
@@ -272,7 +272,7 @@ def matrix_norm(
     /,
     *,
     keepdims: bool = False,
-    ord: Optional[Union[int, float, Literal[inf, -inf, "fro", "nuc"]]] = "fro",
+    ord: Optional[Union[int, float, Literal[inf, -inf, "fro", "nuc"]]] = "fro",  # type: ignore
```

Comment:
Note that there's currently no way to express literal floats in the type system, hence the need for the `# type: ignore` comment. Could maybe consider adding two `Inf` and `NInf` types (or something along those lines) and declare them as `float` aliases?
``` python
Inf = float
NInf = float
```

Reply:
I agree it's annoying Python can't do `Literal[<float>]`. I considered doing this, but I think I prefer not to change the spec in this PR. Changing to alias values will require additional docs and explanation since `Inf = float` is not the same as `math.inf`, and the latter is part of the spec.

Reply:
>  Changing to alias values will require additional docs and explanation since Inf = float is not the same as math.inf, and the latter is part of the spec.

True, though the `float` already being present as a valid `ord` parameter means that the inclusion of `math.inf` is redundant either way (from a typing perspective at least), so to me this doesn't sound like such a change would violate the spec.

Reply:
✅

Reply:
I added the type ignores only to make my local mypy happier (mypy isn't part of the the CI on this repo). 
I think making these changes should be done in a followup PR. I'm happy to make the changes in another PR.